### PR TITLE
C2c source target v1 to v2

### PR DESCRIPTION
**Pull request by vishnu-vasan**
## Issue ticket number and link
Closes #xxxx (Replace xxxx with the GitHub issue number)

## Checklist before requesting a review
- [ Yes ] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [ ] CI Tests succeed and look good!

## Requirements after merging

- [ ] Need to regenerate `common/` resources

- [ ] Need to regenerate task-specific resources. Specify: <all or ...>

### CODE REVIEW

File: src/common/create_task_readme/render_all.sh [hunk 1]
```diff
@@ -5,4 +5,4 @@ TASK_IDS=`ls src/tasks`
 for task_id in $TASK_IDS; do
   echo ">> Processing $task_id"
   viash run src/common/create_task_readme/config.vsh.yaml -- --task $task_id
-done
```
Comment (rcannood) [inline src/common/create_task_readme/render_all.sh:8]: Please undo this change

```suggestion
done

```

File: src/tasks/cell_cell_communication_source_target/api/file_solution.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        description: Long description of the dataset.
+        required: true
+      - name: dataset_organism
+        type: string
+        description: The organism of the sample in the dataset.
+        required: false
+      - type: string
+        name: ccc_target
+        description: assumed truth value. A pandas data frame containing response, source, and target
+        required: true
```
Comment (rcannood) [inline src/tasks/cell_cell_communication_source_target/api/file_solution.yaml:58]: * Please detail exactly what information is inside the data frame and what it represents. For example:
  - `source`: The cell type that initiates the communication signal.
  - `target`: The cell type which is intended to receive the communicational signal.
  - `ligand`: The gene of the signalling molecule released by the source cell.
  - `receptor`: The gene of the protein on the surface of the target cell that the ligand specifically binds to.
* This is not a string. Perhaps `type: list` or `type: dataframe` is more suitable. This, however, is the first case we need to store a data frame in the uns, so we'll have to add this structure. **Alternatively**: it would be possible to simply add the different columns of the `ccc_target` directly to the `uns` itself, at least for now.

what do you think of something like this?
```yaml
      - type: string
        name: ccc_source
        description: The cell type that initiates the communication signal. This serves as the ground-truth cell-cell communication (CCC) information. This vector contains one value per CCC interaction.
        required: true
      - type: string
        name: ccc_target
        description: The cell type which is intended to receive the communicational signal. This vector needs to be of the same length as `ccc_source`.
        required: true
      - type: string
        name: ccc_ligand
        description: The gene of the signalling molecule released by the source cell. This vector needs to be of the same length as `ccc_source`.
        required: true
      - type: string
        name: ccc_receptor
        description: The gene of the protein on the surface of the target cell that the ligand specifically binds to. This vector needs to be of the same length as `ccc_source`.
        required: true
```

Alternatively, something like this would also be possible, but then we need to add this data type to the openproblems_v2 helper functions:

```yaml
      - type: dataframe
        name: ccc_groundtruth
        description: A set of cell-cell communication (CCC) interactions. This data frame contains one row per CCC interaction.
        columns:
          - type: string
            name: source
            description: The cell type that initiates the communication signal.
            required: true
          - type: string
            name: target
            description: The cell type which is intended to receive the communicational signal.
            required: true
          - type: string
            name: ligand
            description: The gene of the signalling molecule released by the source cell.
            required: true
          - type: string
            name: receptor
            description: The gene of the protein on the surface of the target cell that the ligand specifically binds to.
            required: true
```

I think the second option would be the best, but again, some code in the `src/common/` folder would need to be updated for this to function as intended.

What do you think?

File: src/tasks/cell_cell_communication_source_target/process_dataset/config.vsh.yaml
Comment (rcannood) [inline src/tasks/cell_cell_communication_source_target/process_dataset/config.vsh.yaml:1]: Don't add empty files as  viash components, as this will cause the CI to flag them as broken.

Comment (vishnu-vasan) [reply src/tasks/cell_cell_communication_source_target/process_dataset/config.vsh.yaml:1]: yeah sure! sorry, i didnt notice that these files were tracked/added when i made the push

File: src/tasks/cell_cell_communication_source_target/api/file_solution.yaml (hunk 2 above)
Comment (vishnu-vasan) [reply src/tasks/cell_cell_communication_source_target/api/file_solution.yaml:58]: Hey Robrecht! 
Sure, yeah i can see that this will be the first of its type as being a df. Even i agree that the second choice looks quite neater and like a best fit. If you think updating the code in the `src/common` directory wont be complicated, we can go for this. Can you tell me what has to be edited over there?

File: src/common/create_task_readme/render_all.sh (hunk 1 above)
Comment (vishnu-vasan) [reply src/common/create_task_readme/render_all.sh:8]: noted

File: src/tasks/cell_cell_communication_source_target/api/file_solution.yaml (hunk 2 above)
Comment (rcannood) [reply src/tasks/cell_cell_communication_source_target/api/file_solution.yaml:58]: Could you start off with the first approach for the time being? Because I'm away for little over a week :) Let's keep the `type: dataframe` in mind for when I get back.

Comment (vishnu-vasan) [reply src/tasks/cell_cell_communication_source_target/api/file_solution.yaml:58]: Sure!

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  --dataset_name "Mouse Brain Atlas" \
+  --dataset_url "http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71585" \
+  --dataset_reference "tasic2016adult" \
+  --dataset_summary "Adult mouse primary visual cortex" \
+  --dataset_description "A murine brain atlas with adjacent cell types as assumed benchmark truth, inferred from deconvolution proportion correlations using matching 10x Visium slides (see Dimitrov et al., 2022)." \
+  --dataset_organism "mus_musculus" \
+  --seed 123 \
+  --do_subsample true \
+  --output_raw '$id/raw.h5ad' \
+  --output_dataset '$id/dataset.h5ad' \
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:40]:
```suggestion
  --output_normalized '$id/normalized.h5ad' \
  --output_hvg '$id/hvg.h5ad' \
  --output_pca '$id/pca.h5ad' \
  --output_knn '$id/knn.h5ad' \
```

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  --obs_batch "tech" \
+  --var_feature_name "index" \
+  --layer_counts "counts" \
+  --dataset_name "Mouse Brain Atlas" \
+  --dataset_url "http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71585" \
+  --dataset_reference "tasic2016adult" \
+  --dataset_summary "Adult mouse primary visual cortex" \
+  --dataset_description "A murine brain atlas with adjacent cell types as assumed benchmark truth, inferred from deconvolution proportion correlations using matching 10x Visium slides (see Dimitrov et al., 2022)." \
+  --dataset_organism "mus_musculus" \
+  --seed 123 \
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:37]:
```suggestion
  --normalization_methods log_cp10k \
```

File: src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  summary: "The dataset to pass to a method."
+  slots:
+    layers: 
+      - type: integer
+        name: counts
+        description: Raw counts
+        required: true
+    obs:
+      - type: string
+        name: label
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml:14]:
```suggestion
        name: cell_type
```
see https://vscode.dev/github/vishnu-vasan/openproblems-v2/blob/c2c_source_target/src/datasets/api/file_raw.yaml#L33

File: src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        description: Long description of the dataset.
+        required: true
+      - name: dataset_organism
+        type: string
+        description: The organism of the sample in the dataset.
+        required: false
+      - type: string
+        name: ccc_target
+        description: The cell type which is intended to receive the communicational signal. This vector needs to be of the same length as `ccc_source`.
+        required: true
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml:49]: You require the `ccc_target` in `uns` of the dataset but it is not in the dataset loader:
https://github.com/vishnu-vasan/openproblems-v2/blob/eaac6f6fc6b6d619a268bd6b7305752fa4dfba24/src/datasets/api/file_raw.yaml#L175

Add it also here but do not set it as required as this is not required for all datasets and tasks.

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 7]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# get the root of the directory
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# ensure that the command below is run from the root of the repository
+cd "$REPO_ROOT"
+
+DATASET_DIR=resources_test/cell_cell_communication_source_target/mouse_brain_atlas
+
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:10]:
```suggestion
DATASET_DIR=resources_test/common
```

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+set -e
+
+mkdir -p $DATASET_DIR
+
+# wget https://raw.githubusercontent.com/theislab/scib/c993ffd9ccc84ae0b1681928722ed21985fb91d1/scib/resources/g2m_genes_tirosh_hm.txt -O $DATASET_DIR/temp_g2m_genes_tirosh_hm.txt
+# wget https://raw.githubusercontent.com/theislab/scib/c993ffd9ccc84ae0b1681928722ed21985fb91d1/scib/resources/s_genes_tirosh_hm.txt -O $DATASET_DIR/temp_s_genes_tirosh_hm.txt
+# KEEP_FEATURES=`cat $DATASET_DIR/temp_g2m_genes_tirosh_hm.txt $DATASET_DIR/temp_s_genes_tirosh_hm.txt | paste -sd ":" -`
+
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:18]:
```suggestion

```

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  --output_raw '$id/raw.h5ad' \
+  --output_dataset '$id/dataset.h5ad' \
+  --output_meta '$id/dataset_meta.yaml' \
+  --output_state '$id/state.yaml' \
+  --publish_dir "$DATASET_DIR"
+
+# rm -r $DATASET_DIR/temp_*
+
+# # run task process dataset components
+src/tasks/cell_cell_communication_source_target/resources_test_scripts/mouse_brain_atlas.sh
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:48]:
```suggestion
# run task process dataset components
```

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 10]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# get the root of the directory
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# ensure that the command below is run from the root of the repository
+cd "$REPO_ROOT"
+
+DATASET_DIR=resources_test/cell_cell_communication_source_target/mouse_brain_atlas
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:9]:
```suggestion
DATASET_DIR=resources_test/common
```

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+set -e
+
+mkdir -p $DATASET_DIR
+
+# wget https://raw.githubusercontent.com/theislab/scib/c993ffd9ccc84ae0b1681928722ed21985fb91d1/scib/resources/g2m_genes_tirosh_hm.txt -O $DATASET_DIR/temp_g2m_genes_tirosh_hm.txt
+# wget https://raw.githubusercontent.com/theislab/scib/c993ffd9ccc84ae0b1681928722ed21985fb91d1/scib/resources/s_genes_tirosh_hm.txt -O $DATASET_DIR/temp_s_genes_tirosh_hm.txt
+# KEEP_FEATURES=`cat $DATASET_DIR/temp_g2m_genes_tirosh_hm.txt $DATASET_DIR/temp_s_genes_tirosh_hm.txt | paste -sd ":" -`
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:17]:
```suggestion
```

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+  --layer_counts "counts" \
+  --dataset_name "Mouse Brain Atlas" \
+  --dataset_url "http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71585" \
+  --dataset_reference "tasic2016adult" \
+  --dataset_summary "Adult mouse primary visual cortex" \
+  --dataset_description "A murine brain atlas with adjacent cell types as assumed benchmark truth, inferred from deconvolution proportion correlations using matching 10x Visium slides (see Dimitrov et al., 2022)." \
+  --dataset_organism "mus_musculus" \
+  --seed 123 \
+  --do_subsample true \
+  --output_raw '$id/raw.h5ad' \
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:39]:
```suggestion
  --output_raw  force_null \
  --output_normalized force_null \
  --output_hvg force_null \
  --output_pca force_null \
  --output_knn force_null \

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+  --output_dataset '$id/dataset.h5ad' \
+  --output_meta '$id/dataset_meta.yaml' \
+  --output_state '$id/state.yaml' \
+  --publish_dir "$DATASET_DIR"
+
+# rm -r $DATASET_DIR/temp_*
+
+# # run task process dataset components
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:47]:
```suggestion
# run task process dataset components
```

File: src/tasks/cell_cell_communication_source_target/resources_test_scripts/mouse_brain_atlas.sh [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+# get the root of the directory
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# ensure that the command below is run from the root of the repository
+cd "$REPO_ROOT"
+
+set -e
+
+RAW_DATA=resources_test/common
+DATASET_DIR=resources_test/cell_cell_communication_source_target/mouse_brain_atlas
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/resources_test_scripts/mouse_brain_atlas.sh:14]:
```suggestion
DATASET_DIR=resources_test/cell_cell_communication_source_target
```

File: src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml (hunk 6 above)
Comment (vishnu-vasan) [reply src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml:49]: Got you, i will add it over there

File: src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+      - name: dataset_description
+        type: string
+        description: Long description of the dataset.
+        required: true
+      - name: dataset_organism
+        type: string
+        description: The organism of the sample in the dataset.
+        required: false
+      - type: string
```
[the remark this answers is not in the document]
Comment (vishnu-vasan) [reply src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml]: The individual fields of the ccc_source slot are all strings only, why are we changing this to object?

[the remark this answers is not in the document]
Comment (vishnu-vasan) [reply src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml]: Okay i misunderstood, thanks for correcting. Will change it across the other files and push it.

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:1]: change fn to`allen_brain_atlas.sh`

File: src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+  --output_knn force_null \
+  --output_dataset '$id/dataset.h5ad' \
+  --output_meta '$id/dataset_meta.yaml' \
+  --output_state '$id/state.yaml' \
+  --publish_dir "$DATASET_DIR"
+
+# rm -r $DATASET_DIR/temp_*
+
+# run task process dataset components
+src/tasks/cell_cell_communication_source_target/resources_test_scripts/mouse_brain_atlas.sh
```
Comment (KaiWaldrant) [inline src/datasets/resource_test_scripts/allen_mouse_brain_atlas.sh:50]:
```suggestion
src/tasks/cell_cell_communication_source_target/resources_test_scripts/allen_brain_atlas.sh
```

File: src/tasks/cell_cell_communication_source_target/api/comp_control_method.yaml [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+      __merge__: file_solution.yaml
+      direction: input
+      required: true
+    - name: "--output"
+      __merge__: file_prediction.yaml
+      direction: output
+      required: true
+  test_resources:
+    - path: /resources_test/cell_cell_communication_source_target/mouse_brain_atlas
+      dest: resources_test/cell_cell_communication_source_target/mouse_brain_atlas
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/comp_control_method.yaml:34]:
```suggestion
    - path: /resources_test/cell_cell_communication_source_target/allen_brain_atlas
      dest: resources_test/cell_cell_communication_source_target/allen_brain_atlas
```

File: src/tasks/cell_cell_communication_source_target/api/comp_method.yaml [hunk 18]
```diff
@@ -0,0 +1,30 @@
...
+      __merge__: file_test.yaml
+      direction: input
+      required: true
+    - name: "--output"
+      __merge__: file_prediction.yaml
+      direction: output
+      required: true
+  test_resources:
+    - path: /resources_test/cell_cell_communication_source_target/mouse_brain_atlas
+      dest: resources_test/cell_cell_communication_source_target/mouse_brain_atlas
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/comp_method.yaml:25]:
```suggestion
    - path: /resources_test/cell_cell_communication_source_target/allen_brain_atlas
      dest: resources_test/cell_cell_communication_source_target/allen_brain_atlas
```

File: src/tasks/cell_cell_communication_source_target/api/comp_metric.yaml [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+      __merge__: file_prediction.yaml
+      direction: input
+      required: true
+    - name: "--output"
+      __merge__: file_score.yaml
+      required: true
+      direction: output
+  test_resources:
+    - path: /resources_test/cell_cell_communication_source_target/mouse_brain_atlas
+      dest: resources_test/cell_cell_communication_source_target/mouse_brain_atlas
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/comp_metric.yaml:25]:
```suggestion
    - path: /resources_test/cell_cell_communication_source_target/allen_brain_atlas
      dest: resources_test/cell_cell_communication_source_target/allen_brain_atlas
```

File: src/tasks/cell_cell_communication_source_target/api/comp_process_dataset.yaml [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+      __merge__: file_test.yaml
+      direction: output
+      required: true
+    - name: "--output_solution"
+      __merge__: file_solution.yaml
+      direction: output
+      required: true
+  test_resources:
+    - path: /resources_test/cell_cell_communication_source_target/mouse_brain_atlas
+      dest: resources_test/cell_cell_communication_source_target/mouse_brain_atlas
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/comp_process_dataset.yaml:29]:
```suggestion
    - path: /resources_test/cell_cell_communication_source_target/allen_brain_atlas
      dest: resources_test/cell_cell_communication_source_target/allen_brain_atlas
```

File: src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml [hunk 21]
```diff
@@ -0,0 +1,49 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/dataset.h5ad"
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_common_dataset.yaml:2]:
```suggestion
example: "resources_test/cell_cell_communication_source_target/allen_brain_atlas/dataset.h5ad"
```

File: src/tasks/cell_cell_communication_source_target/api/file_prediction.yaml [hunk 22]
```diff
@@ -0,0 +1,24 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/prediction.h5ad"
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_prediction.yaml:2]:
```suggestion
example: "resources_test/cell_cell_communication_source_target/allen_brain_atlas/dataset.h5ad"
```

File: src/tasks/cell_cell_communication_source_target/api/file_score.yaml [hunk 23]
```diff
@@ -0,0 +1,25 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/score.h5ad"
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_score.yaml:2]:
```suggestion
example: "resources_test/cell_cell_communication_source_target/allen_brain_atlas/dataset.h5ad"
```

File: src/tasks/cell_cell_communication_source_target/api/file_solution.yaml [hunk 24]
```diff
@@ -0,0 +1,61 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/solution.h5ad"
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_solution.yaml:2]:
```suggestion
example: "resources_test/cell_cell_communication_source_target/allen_brain_atlas/dataset.h5ad"
```

File: src/tasks/cell_cell_communication_source_target/api/file_test.yaml [hunk 25]
```diff
@@ -0,0 +1,37 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/test.h5ad"
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_test.yaml:2]:
```suggestion
example: "resources_test/cell_cell_communication_source_target/allen_brain_atlas/dataset.h5ad"
```

File: src/tasks/cell_cell_communication_source_target/api/file_train.yaml [hunk 26]
```diff
@@ -0,0 +1,37 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/mouse_brain_atlas/train.h5ad"
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/api/file_train.yaml:2]:
```suggestion
example: "resources_test/cell_cell_communication_source_target/allen_brain_atlas/dataset.h5ad"
```

File: src/tasks/cell_cell_communication_source_target/process_dataset/script.py [hunk 27]
```diff
@@ -0,0 +1,60 @@
+import sys
+import anndata as ad
+# from openproblems.tasks._cell_cell_communication._common.utils import ligand_receptor_resource
+import openproblems as op
+import pandas as pd
+
+## VIASH START
+par = {
+    "input": "resources_test/cell_cell_communication_source_target/mouse_brain_atlas",
```
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/process_dataset/script.py:9]:
```suggestion
    "input": "resources_test/cell_cell_communication_source_target/allen_brain_atlas",
```

File: src/tasks/cell_cell_communication_source_target/resources_test_scripts/mouse_brain_atlas.sh
Comment (KaiWaldrant) [inline src/tasks/cell_cell_communication_source_target/resources_test_scripts/mouse_brain_atlas.sh:1]: Change fn to `allen_brain_atlas.sh`

**Review by KaiWaldrant — COMMENTED** [review]
Make sure you use `allen_brain_atlas` instead of `allen_mouse_brain_atlas` or `mouse_brain_atlas`

**Comment by vishnu-vasan** [comment]
Thanks Kai, i will change them

**Comment by rcannood** [comment]
This work has been moved to https://github.com/openproblems-bio/task_cell_cell_communication! :) Closing this PR.
